File: Swift/Lista_04_EX04/UsuarioGroup.swift
import Foundation
 
 public class UsuarioGroup : Usuario{
 
 	override func descricao() {
        print("O usuario Group, pode add e remover usuarios de grupos")
		print("Esse usuario Group poderá apagar usuários de seu grupo")
    }
	
	func despedidaGroup( ) {
		print("Até logo \(login)!")
	}
	
}

File: Java/Lista 4/L4_Ex1.4/src/l4_ex1/pkg4/UsuarioGroup.java

package l4_ex1.pkg4;

public class UsuarioGroup extends Usuario{
    
    public UsuarioGroup(String login) {
        super(login);
    }
    
    @Override
    public void descricao(){
        System.out.println("O usuario Group, pode add e remover usuarios de grupos");
        System.out.println(" Esse usuario Group poderá apagar usuários de seu grupo");
    }
           
    public void despedidaGroup(){
        System.out.println("Até logo " + this.getLogin() + "!");
    }
    
}
